client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,10 @@
+const NavBar = () => {
```

Comment:
For functional React components, I think I prefer function syntax vs const/fat-arrow syntax. I.e. between:

```
function NavBar() {
```
and
```
const NavBar = () => {
```

I think it's a little less busy to use `function` and matches the style of the template App component...